src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java

```diff
@@ -285,32 +285,12 @@ public boolean isAnnotated(TypeMirror e) {
         return !e.getAnnotationMirrors().isEmpty();
     }
 
-    public boolean isAnnotationType(Element e) {
-        return new SimpleElementVisitor14<Boolean, Void>() {
-            @Override
-            public Boolean visitExecutable(ExecutableElement e, Void p) {
-                return visit(e.getEnclosingElement());
-            }
-
```

Comment:
do you know/understand why this code was there before?

Reply:
I don't know why this code was added there in the first place. What I do know is that this code is never called from `make docs` or our tests.

Reply:
Ah! I overcomplicated things; I've forgotten what I did in this PR already. Look at https://github.com/openjdk/jdk/commit/75120aa205bf3cad614fd4e67d5bb13e20f44a1e#diff-b2fe0a536cfed983d5ae178345c2a3433aa5acb2092d3d06ddc5dd0541fbdbe1L272 and note the commit message:

> Fixes confusing implementation that accepted elements enclosed by an annotations interface; **adjusts some of the call sites**

It looks like that it might've been the only legitimate use case pushed down to this method. It's gone now anyway.

Reply:
All noted and understood.